yaas/bootstrap.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+
+
+class Bootstrap:
+
+    """ Interact with agent bootstrap requests. """
+
+    def __init__(self, client):
+        self.client = client
+
+    def agents(self, hosts, sshKey, user, userRunAs=None):
+        """ Install and configure ambari-agent on hosts. """
+        req = {
+            'hosts': hosts,
+            'sshKey': sshKey,
+            'user': user,
+            }
+        if userRunAs is not None:
+            req['userRunAs'] = userRunAs
+        response = self.client.request(
+            'post',
+            '/api/v1/bootstrap',
+            json=req)
+        return response.json()
```

Comment:
E.g. I use import yaas to export a blueprint. I want that to be a blueprint that I can later post to the server. I feel like the hrefs are only meant to show you what endpoints you can expand. They're not important to a user of yaas.